.github/workflows/dockerd.yml

```diff
@@ -12,7 +12,7 @@ on:
 env:
   SETUP_BUILDX_VERSION: "latest"
   SETUP_BUILDKIT_IMAGE: "moby/buildkit:latest"
-  TESTFLAGS: "-v --parallel=1 --timeout=30m"
+  TESTFLAGS: "-v --parallel=6 --timeout=30m"
```

Comment:
I looked at your pipeline run on your fork https://github.com/jedevc/buildkit/actions/runs/5520028773

Seems many tests failed but not sure if linked to //. Trigered one on this repo to look at the state: https://github.com/moby/buildkit/actions/runs/5521394187

Reply:
Can you also create a new branch from `v0.11`, vendor in moby and open draft PR there to see if everything looks good?

Reply:
See https://github.com/moby/moby/pull/45936.